x/evm/types/storage.go

```diff
@@ -5,27 +5,20 @@ import (
 	"encoding/json"
 	"fmt"
 
-	sdkerrors "github.com/okex/exchain/libs/cosmos-sdk/types/errors"
 	ethcmn "github.com/ethereum/go-ethereum/common"
+	sdkerrors "github.com/okex/exchain/libs/cosmos-sdk/types/errors"
 )
 
 // Storage represents the account Storage map as a slice of single key value
 // State pairs. This is to prevent non determinism at genesis initialization or export.
-type Storage []State
+type Storage map[ethcmn.Hash]State
```

Comment:
be careful here : https://github.com/okex/exchain/blob/417d97dfcf005a3fbf83fd1a79de330741b5017b/x/evm/types/genesis.go#L30